src/main/java/org/killbill/billing/entitlement/api/SubscriptionApi.java

```diff
@@ -159,6 +160,21 @@ public interface SubscriptionApi extends KillbillApi {
     @RequiresPermissions(ENTITLEMENT_CAN_PAUSE_RESUME)
     public void addBlockingState(BlockingState blockingState, LocalDate effectiveDate, Iterable<PluginProperty> properties, CallContext context)
             throws EntitlementApiException;
+    
+    /**
+     * Add a <code>BlockingState</code>
+     * <p/>
+     * The date is interpreted by the system to be in the timezone specified at the <code>Account</code>
```

Comment:
Nit: there is no _interpretation_, since the `DateTime` supplied will have a timezone.